decentraland/unity-explorer
# fix: chat spam

## What does this PR change?

- Re-applied the fix from https://github.com/decentraland/unity-explorer/pull/701
- Updated protocols to @next version

## How to test the changes?

- Go to the same place in unity-renderer and this build with different accounts.
- Spam the chat
- The chats should be different for both clients and no message from one should appear on the other

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
